tests/unit_tests/c_wrappers/res/enkf/test_run_path_creation.py

```diff
@@ -267,3 +267,31 @@ def test_that_data_file_sets_num_cpu(eclipse_data, expected_cpus):
     res_config = ResConfig("config.ert")
     ert = EnKFMain(res_config)
     assert int(ert.get_context()["<NUM_CPU>"]) == expected_cpus
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+def test_that_runpath_substitution_remain_valid():
+    """
+    This checks that runpath substitution remain intact.
+    """
+    config_text = dedent(
+        """
+        NUM_REALIZATIONS 2
+        JOBNAME my_case%d
+        RUNPATH real_%d/iter_%d
```

Comment:
Suggest to change to `realization-%d/iter-%d` to be consistent with other files (real_%d.. is legacy)

Reply:
Added!